### PR TITLE
Feature/export participants + bugfix

### LINKED ISSUE #5: Overzicht inschrijvingen per schema
In schema zelf, of aparte pagina?

### PR DESCRIPTION
**Pull request by kliptonize**
Exporting participants to csv is now possible! 👏 
@EveraertJan also changed some stuff to allow for multiple participations to be added at once, without mistakes.

Fixes #5

### CODE REVIEW

**Comment by EveraertJan** [comment]
@kliptonize wanna check?

File: images/api/src/Answers.js [hunk 1]
```diff
@@ -51,8 +51,8 @@ class Answers {
       // add to answers, if there isn't already a 417 underway
       if(insert.length > 0 && !res.headersSent){
         //Up counter for each insertion
-        for(var i=0;i<insert.length;i++){
-          await pg.update({'current': pg.raw('current + 1')}).table('cells').where({uuid: insert[i]['cellID']}).then(function(v) {
+        for(let i=0;i<insert.length;i++){
+          await pg.update({'current': pg.raw('current + 1')}).table('cells').where({uuid: insert[i]['cellID']}).then((v) {
```
Comment (kliptonize) [inline images/api/src/Answers.js]: Werkt dat? Moet daar geen `(v) => {` staan?

**Comment by kliptonize** [comment]
Is er verder iets aangepast eigelijk, buiten `const`, de ' ipv " en function definitions?

**Comment by EveraertJan** [comment]
nieje, daarmee :D

**Pull request merged**